packages/api-client/src/odsql/index.ts

```diff
@@ -168,6 +168,17 @@ export const string = (value: string) => JSON.stringify(value);
 
 export const dateTime = (date: Date) => `date'${date.toISOString()}'`;
 
+/**
+ * ODSQL function to normalize field name.
+ *
+ * - Can be used for `order_by` and `group_by` clauses.
+ * - Example usage: `select=count(*) & group_by=lower(artist)`.
+ *
+ * @param {string} fieldName - The field name to normalize.
+ * @returns {string}  The lower ODSQL function.
+ */
+export const lower = (fieldName: string) => `lower(${fieldName})`;
```

Comment:
We could include the result of the `field()` function
```suggestion
export const lower = (fieldName: string) => field(`lower(${fieldName})`);
```